src/auto-evo/RunResults.cs

```diff
@@ -7,6 +7,9 @@
 using System.Linq;
 using Godot;
 using JetBrains.Annotations;
+using PossibleSpecies =
+    System.Tuple<Species, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<Patch, long>>,
```

Comment:
Couldn't this be a named tuple that would make the code in [RegisterNewSpecies.cs](https://github.com/Revolutionary-Games/Thrive/pull/5388/files#diff-773e52f10083a2dbed39bb1156158836e8e8e94b219d9e74dc54fa238fc9407d) much easier to follow?

Reply:
C# won't let me type alias that. I could turn it into a struct and do it that way.

Reply:
I think that would be better. In the first place needing to have this using to make things readable kind of indicates to me that this is pretty crazy amount of parameters to pack together like this in the places this is used.